packages/@angular/cli/tasks/eject.ts

```diff
@@ -554,6 +554,16 @@ export default Task.extend({
         packageJson['scripts']['pree2e'] = pree2eNpmScript;
         packageJson['scripts']['e2e'] = 'protractor ./protractor.conf.js';
 
+        if (runTaskOptions.environment === 'production' && usesServiceWorker(project.root)
+            && !!runTaskOptions.serviceWorker) {
```

Comment:
I don't think we have a option for service worker proper. It's just a flag in `.angular-cli.json`.

Reply:
You should be able to get it as `appConfig.serviceWorker`

Reply:
I added both checks.